lib/src/component/resize_sensor.dart

```diff
@@ -44,10 +73,20 @@ abstract class ResizeSensorPropsMixin {
 
   Map get props;
 
-  /// A function invoked when the resize sensor is initialized.
+  /// A function invoked with a [ResizeSensorEvent] argument when the resize sensor is initialized.
+  ///
+  /// > Will never be called if [quickMount] is `true`.
+  ///
+  /// Related: [onResize]
   ResizeSensorHandler onInitialize;
 
-  /// A function invoked when the parent element is resized.
+  /// A function invoked with a [ResizeSensorEvent] argument when the `children` of the
```

Comment:
#nit `ResizeSensorEvent` is a redundant type reference; it's implied in the `ResizeSensorHandler` typing.